docs/synthetics/ci-cd-integration.md

```diff
@@ -0,0 +1,300 @@
+title: CI/CD Integration
+description: Guide to how to run HTTP/Browser monitors from your CI/CD pipeline.
+
+In addition to the scheduled monitor runs, you can also trigger monitor runs using API. You can use this API to trigger monitors runs as part of your CI/CD pipeline and block the deployment if the runs fail. On failure of the run, you will be alerted on the configured alert [notification hooks](../alerts/alert-notifications.md). The run monitor API can be used to:
+
+* Test the APIs, websites, and the user journeys in your PR and staging environments and annotate the pull requests with the results. 
+* Test the APIs, websites, and the user journeys in production immediately after deployment and alert when it fails.
+* Track and catch major changes in website metrics like page load time, page size, request count, 3rd Party API performance, etc introduced as part of code changes.
+
+## Run Monitor API
+
+The run monitor API can be triggered by sending an HTTP request with the below configuration:
+
+**US Region Endpoint** - `https://apps.sematext.com/synthetics-api/api/v3/apps/<appId>/monitors/runs`
+
+**EU Region Endpoint** - `https://apps.sematext.com/synthetics-api/api/v3/apps/<appId>/monitors/runs`
+
+**HTTP Method** - `POST`
+
+**Request Headers** - `Authorization: apiKey <apiKey>`
+
+**Request Body**
+```json
+[
+    {
+        "monitorId": <monitorId-1>,
+        "locations": [<locationId-1>, <locationId-2>,...],
+    },
+    {
+        "monitorId": <monitorId-2>,
+        "locations": [<locationId-1>, <locationId-2>,...],
+    },
+    ...
+]
+```
+
+* The `<appId>` and `<monitorId>` values can be extracted from the URL of the Monitor Overview page in UI. For example, if the Monitor Overview page URL is `https://apps.sematext.com/ui/synthetics/12345/monitors/276` then the `appId` is `12345` and `monitorId` is `276`.
+* `<apiKey>` of your account can be copied from Settings -> API page.
+* `<locationId>` - List of locations to run the monitor from. If not specified, the monitor will be run from all locations specified in the monitor configuration. The supported locations are:
```

Comment:
I think we already have a list of locations somewhere.  Consider linking there to be more DRY.

Reply:
Yes. We have it on the overview page. I also wanted to add the region-id there. Currently, that table is organized by continents. I cannot find a way to keep the continent wise grouping and add region id. So I added it here.